src/components/MediaPlayer/VideoJS/components/js/VideoJSProgress.js

```diff
@@ -240,6 +240,19 @@ function ProgressBar({
     const left = canvasTargets.filter((_, index) => index < srcIndex);
     setTRight(right);
     setTLeft(left);
+
+    // Update progressRef on keydown events for left/right arrows
+    document.addEventListener('keydown', (event) => {
+      let pressedKey = event.which;
+      if (pressedKey === 37) {
+        event.preventDefault();
+        setProgress(player.currentTime() - 5);
+      }
+      if (pressedKey === 39) {
+        event.preventDefault();
+        setProgress(player.currentTime() + 5);
+      }
+    });
```

Comment:
How does this interact with the left/right arrow handling in the playerHotKeys function? Depending on what order the functions fire in, this seems like it could potentially cause the `progressRef` to be offset by 5 seconds.

Reply:
`progressRef` variable is used only to update the progress bar UI. While any updates to `player.currentTime()` update `progressRef` variable it doesn't happen the other way around. So it wouldn't offset the time by 5 seconds. 

However, this is only relevant to Safari because the `timeUpdateHandler` function skips this update for Safari.
Maybe this eventListener can be scoped further? Let me take look at that again.

Reply:
You were right, it does offset the time. Thank you for catching it 🎉 
As it turns out we don't need that eventListener at all!